falcon/falcon3-10B-trt-llm-spec-dec/config.yaml

```diff
@@ -0,0 +1,48 @@
+model_metadata:
+  tags:
+    - openai-compatible
+  example_model_input:
+    frequency_penalty: 1
+    max_tokens: 512
+    messages:
+      - content: You are a knowledgable, engaging, history teacher.
+        role: system
+      - content: What was the role of falcons in Lord of the Rings?
+        role: user
+    stream: true
+    temperature: 0.3
+    top_k: 40
+    top_p: 1.0
+  repo_id: tiiuae/Falcon3-10B-Instruct
+model_name: Falcon3-10B-Instruct-with-1B-SpecDec
+resources:
+  accelerator: H100
+  cpu: "1"
+  memory: 24Gi
+  use_gpu: true
+trt_llm:
+  build:
+    base_model: llama
+    checkpoint_repository:
+      repo: tiiuae/Falcon3-10B-Instruct
+      source: HF
+    max_seq_len: 8192
```

Comment:
Should this match the runtime tokens?

Reply:
This should match: https://huggingface.co/tiiuae/Falcon3-1B-Instruct/blob/28ba2251970a01dd1edc7ba7dad2eb71216ccfdf/config.json#L13